sway/commands/include_one.c

```diff
@@ -0,0 +1,78 @@
+#define _XOPEN_SOURCE 700 
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "list.h"
+#include "log.h"
+#include "unistd.h"
+#include <string.h>
+#include <libgen.h>
+#include <wordexp.h>
+
+typedef struct {
+	char *name;
+	char *path;
+} config_location_map;
+
+int compare_paths(const config_location_map* item, char* abspath) {
+	if (strcmp(item->name, basename(abspath))==0) {
```

Comment:
There are two versions of basename, did you check this? They have different semantics.